src/System.Net.Security/src/System/Net/Security/SslStreamPal.OSX.cs

```diff
@@ -95,10 +94,11 @@ public static void VerifyPackageInfo()
 
                 unsafe
                 {
-                    fixed (byte* offsetInput = &input[offset])
+                    MemoryHandle memHandle = input.Retain(true);
```

Comment:
Nit: for these Retain calls, it'd be helpful to me if the bool arg was named, e.g. `Retain(pin: true)`

Reply:
done added `Retain(pin: true)`